src/Altinn.Platform/Altinn.Platform.Storage/Storage/Startup.cs

```diff
@@ -78,20 +88,19 @@ public void ConfigureServices(IServiceCollection services)
                     .Build();
                 config.Filters.Add(new AuthorizeFilter(policy));
             }).AddNewtonsoftJson();
+
             services.Configure<AzureCosmosSettings>(Configuration.GetSection("AzureCosmosSettings"));
             services.Configure<AzureStorageConfiguration>(Configuration.GetSection("AzureStorageConfiguration"));
             services.Configure<GeneralSettings>(Configuration.GetSection("GeneralSettings"));
             services.Configure<Common.PEP.Configuration.PepSettings>(Configuration.GetSection("PepSettings"));
             services.Configure<Common.PEP.Configuration.PlatformSettings>(Configuration.GetSection("PlatformSettings"));
 
             GeneralSettings generalSettings = Configuration.GetSection("GeneralSettings").Get<GeneralSettings>();
-
+            
             services.AddAuthentication(JwtCookieDefaults.AuthenticationScheme)
                 .AddJwtCookie(JwtCookieDefaults.AuthenticationScheme, options =>
                 {
-                    options.ExpireTimeSpan = new TimeSpan(0, 30, 0);
-                    options.Cookie.Name = generalSettings.RuntimeCookieName;
```

Comment:
These are no longer required because it is handles by the jwttokenhandler?

Reply:
Correct